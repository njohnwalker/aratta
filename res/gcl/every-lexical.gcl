x := 10 ;
if x == y -> y := y + 1
[] x <= y -> y := 7 - y ;
do v > z -> do od
[] z < v -> if fi
od
[] foo >= baz | ~ true & 0 > 1 -> z := 4 + 30 - (x + y) + 24
[] true -> x,y,z := 10, 20, 76
fi ;
catdog := c
