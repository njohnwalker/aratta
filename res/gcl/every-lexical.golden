@req(false)
x := 10 ;
if x == y ->
      y := y + 1 ;
[] x <= y ->
      y := 7 - y ;
      do @inv()
      [] v > z ->
            do @inv(quux >= ni) od
      [] z < v ->
            if fi
      od
[] (foo >= baz | ~ true) & (0 > 1 | x % 2 == 0) ->
      z := (4 + 30) - ((x * y) / 24) ;
[] true ->
      x, y, z := 10, 20, 76 ;
fi
catdog := c ;
@ens(true)
